File: PYTHON PROGRAMMING/Test37.py
# 37 PRINT SUM OF THOSE NUMBER FROM RANGE1 TO RANGE2 WHICH ARE DIVISIBLE BY THE GIVEN TARGET
def func(limit1, limit2, target):
    sum = 0
    for i in range(limit1, limit2 + 1):
        if(i % target == 0):
            print(i, end=" ")
            sum += i
    print()
    print(f"sum: {sum}")
func(10, 20, 3)
func(30, 50, 4)
func(60, 80, 7)

File: TYPESCRIPT PROGRAMMING/Test37.ts
// 37. PRINT SUM OF THOSE NUMBER FROM RANGE1 TO RANGE2 WHICH ARE DIVISIBLE BY THE GIVEN TARGET
const func37 = (range1: number, range2: number, target: number): void => {
    let sum: number = 0;
    for(let i = range1; i <= range2; i++) {
        if(i % target === 0) {
            sum += i;
        }
    }
    console.log(`Sum: ${sum}`);
}
func37(30, 60, 3)
func37(60, 80, 4)